docs/tfengine/README.md

```diff
@@ -134,8 +136,10 @@ the `terraform` and `terragrunt` binaries to deploy the infrastructure.
     ```
 
 1. Replace the values in a suitable
-   [example](../../examples/tfengine) with values for your infrastructure.
+   [example](../../examples/tfengine) with values for your infrastructure.  
 
+   Project and Bucket names are globally unique and must be changed from defaults.  You will get a 409 error if you have chosen a name that is already taken.
```

Comment:
do we get a 409 error for both cases? project and bucket?

Reply:
I fixed to just say error.  I am sure you get a 409 for bucket but could not find my history for project